spring-integration-core/src/main/java/org/springframework/integration/codec/CompositeCodec.java

```diff
@@ -41,9 +39,20 @@ public class CompositeCodec implements Codec {
 
 	public CompositeCodec(Map<Class<?>, Codec> delegates, Codec defaultCodec) {
 		this.defaultCodec = defaultCodec;
+		Assert.notEmpty(delegates, "delegates must not be empty");
 		this.delegates = new HashMap<Class<?>, Codec>(delegates);
 	}
 
+	/**
+	 * @since 7.0
```

Comment:
This tag means when the API was introduced.
Which is not correct in our case.
And doesn't matter no when it was introduced anyway since we are deprecating it.